packages/url_launcher/url_launcher_web/pubspec.yaml

```diff
@@ -14,31 +11,18 @@ flutter:
         fileName: url_launcher_web.dart
 
 dependencies:
-  url_launcher_platform_interface: ^1.0.9
-  # TODO(mvanbeusekom): Update to use pub.dev once null safety version is published.
-  # url_launcher_platform_interface:
-  #   git:
-  #     url: https://github.com/flutter/plugins.git
-  #     ref: nnbd
-  #     path: packages/url_launcher/url_launcher_platform_interface
+  url_launcher_platform_interface: ^2.0.0-nullsafety
+  meta: ^1.3.0-nullsafety.3
```

Comment:
You could/should drop the `-null-safety.3` bit here! this is stable now

Reply:
Nice! I'm going to give this a shot, I hope this doesn't conflict with other plugins in the repo :P (I think I copied that dep from other plugins just to make sure it was the same!)

Reply:
I've also noticed that pedantic 1.10.0 already has a non -nullsafe version published! Yay!